generic_config_updater/gu_common.py

```diff
@@ -136,6 +139,22 @@ def validate_config_db_config(self, config_db_as_json):
 
         return True, None
 
+    def validate_field_operation(self, old_config, target_config):
+        """
+        Some fields in ConfigDB are restricted and may not allow third-party addition, replacement, or removal. 
+        Because YANG only validates state and not transitions, this method helps to JsonPatch operations/transitions for the specified fields. 
+        """
+        patch = jsonpatch.JsonPatch.from_diff(old_config, target_config)
+        
+        # illegal_operations_to_fields_map['remove'] yields a list of fields for which `remove` is an illegal operation 
+        illegal_operations_to_fields_map = {'add':[],
+                                            'replace': [],
+                                            'remove': ['/PFC_WD/GLOBAL/POLL_INTERVAL']}
```

Comment:
I saw in [sonic-pfcwd.yang](https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-pfcwd.yang#L75) that POLL_INTERVAL will only be valid if its table's key is GLOBAL.

Is it guaranteed that GLOBAL will always have the POLL_INTERVAL field?
In that case, we can add a statement in YANG file if we want to avoid deletion of the POLL_INTERVAL.

Reply:
@neethajohn , can you help confirm if there will ever be a situation where GLOBAL will be present without POLL_INTERVAL field?

@wen587 If this approach is possible, then we must also prohibit removal of GLOBAL. So I think we also need to consider if PFC_WD must always have GLOBAL.

Reply:
@bingwang-ms , Could you help address the question of " if there will ever be a situation where GLOBAL will be present without POLL_INTERVAL field?"

Reply:
> @bingwang-ms , Could you help address the question of " if there will ever be a situation where GLOBAL will be present without POLL_INTERVAL field?"

Yes, I think so. In the `GLOBAL` table, we can also have another entry named `BIG_RED_SWITCH`. If PFC watchdog is not enabled, the `POLL_INTERVAL` can be absent, while the `BIG_RED_SWITCH` can be present. So the `GLOBAL` table is present without `POLL_INTERVAL`.

Reply:
Thank you @bingwang-ms .

In this case, @wen587 , does implementation as-is look alright to you?

Reply:
Hi @bingwang-ms, does that mean `POLL_INTERVAL` could be deleted if PFC watchdog state changes from enable to disable?
Hi @isabelmsft , if Bing confirmed it cannot be deleted. I think it is good to go.

Reply:
Yes, I confirm that.